exporter/exporter.go

```diff
@@ -16,6 +16,7 @@ const metricDashboardTemplate = `
 resource "lightstep_metric_dashboard" "exported_dashboard" {
   project_name = var.project
   dashboard_name = "{{.Attributes.Name}}"
+  dashboard_description = {{escapeQueryString .Attributes.Description}}
```

Comment:
worth renaming this `escapeQueryString` helper? it's just escaping _any_ string that could have linebreaks/other weird stuff in it

Reply:
Makes sense!